keras_cv/custom_ops/box_util.cc

```diff
@@ -269,7 +269,8 @@ double RotatedBox2D::IoU(const RotatedBox2D& other) const {
   return intersection_area / union_area;
 }
 
-bool RotatedBox2D::left_hand_side(const Vertex& point, const Vertex& v1, const Vertex& v2) const {
+bool RotatedBox2D::left_hand_side(const Vertex& point, const Vertex& v1,
```

Comment:
I assume these format changes will be gone from the diff after merging past #1487 -- right?